configs/native.yaml

```diff
@@ -3,7 +3,7 @@ plugin_groups:
   - plugin_group:
       - path: ground_truth_slam/
       - path: gldemo/
-      - path: debugview/
+      # - path: debugview/
```

Comment:
native.yaml can be kept untouched, but since debugview doesn't work with this branch, we may need to fix it first @charmoniumQ We need to modify rt_slam_plugins.yaml to refer to correct version of open_vins (see this PR https://github.com/ILLIXR/open_vins/pull/14) and monado_integration.

Reply:
Yes, let's fix `debugview` first. We'll only change the OpenVINS version in the comment, since it is by default commented out.

Reply:
debugview enabled with the latest commits.